docs: fix grammatical error in transitions.md (#27679)
nuxt/nuxt: docs/1.getting-started/5.transitions.md

## Transition with NuxtPage

> When `<NuxtPage />` is used in `app.vue`, the `transition` prop can be passed directly as a component prop to activate global transition.

Reviewer: ```suggestion
When `<NuxtPage />` is used in `app.vue`, transitions can be configured with the `transition` prop to activate transitions globally.
```
